typescript/cli/src/commands/relayer.ts

```diff
@@ -0,0 +1,37 @@
+import { HyperlaneCore, HyperlaneRelayer } from '@hyperlane-xyz/sdk';
+
+import { CommandModuleWithContext } from '../context/types.js';
+
+import { agentTargetsCommandOption } from './options.js';
+import { MessageOptionsArgTypes } from './send.js';
+
+export const relayerCommand: CommandModuleWithContext<
+  MessageOptionsArgTypes & { chains?: string }
+> = {
+  command: 'relayer',
+  describe: 'Run a CLI relayer',
+  builder: {
+    chains: {
+      ...agentTargetsCommandOption,
+      demandOption: false,
+    },
+  },
+  handler: async ({ context, chains }) => {
+    const chainsArray = chains
+      ? chains.split(',').map((_) => _.trim())
+      : undefined;
```

Comment:
Seems like you meant to set demandOption to true then

Reply:
I think i misunderstood, but its fine for them not to pass it (in which case it runs the relayer everywhere)